source/cpuid/cpuid.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cpuid
+
+import "github.com/klauspost/cpuid"
+
+// CPUIDSource implements FeatureSource.
+type CPUIDSource struct{}
```

Comment:
Could we rename this to just `Source`? That way the qualified name is `cpuid.Source` which sounds a bit better to me.

(Applies to the other feature sources too.)

Reply:
@ConnorDoyle Will do.